### ISSUE TITLE
portal and api blocked after update by api rest

### ISSUE THREAD

**Issue by vdelattre** [gravitee-io/issues#552]
i updated an api with rest api PUT
in my json body i put:
group="RefSite"

i got error 400 with message
{
  "message": "Group [RefSite] can not be found.",
  "http_status": 404
}

but now, all action i made, i have this error.
and i can't delete my api because it's started
and when i want to stop by api, i have the same message
in portal, i can't go on api screen
i logout of portal, and i can't login because same message errror:
![image](https://cloud.githubusercontent.com/assets/22543323/25284066/2af06046-26b6-11e7-91fd-b5f21d2f3684.png)

### Your Environment
* Version used: 1.5.0

**Comment by NicolasGeraud** [comment]
hi @vdelattre could you get api details via curl ?

**Comment by vdelattre** [comment]
No, with curl, i got same error

Le 21 avr. 2017 5:50 PM, "Nicolas Géraud" <notifications@github.com> a
écrit :

**Comment by NicolasGeraud** [comment]
And what about `curl http://MGMT_URL/management/configuration/groups` ?

**Comment by vdelattre** [comment]
The same

Le 21 avr. 2017 7:02 PM, "Nicolas Géraud" <notifications@github.com> a
écrit :

**Comment by NicolasGeraud** [comment]
You mean that if you try to get the list of groups, you've got an error due to the fact that you have updated an API ?
When we retrieve all groups, we don't retrieve informations on apis.
Could you please post your curl and the response ?

What I suspect is that you have update the api group with the group name, not the group id.

Perhaps you could PUT a real groupId to override the old value.

**Comment by brasseld** [comment]
Blocked by: https://github.com/gravitee-io/issues/issues/354

**Comment by vdelattre** [comment]
for information, PUT with real groupId failed for same message

**Comment by NicolasGeraud** [comment]
do you have access to the repository to confirm that your problem is caused by #354 ?

**Comment by vdelattre** [comment]
yes it's same issue

**Comment by NicolasGeraud** [comment]
ok, so as a workaround (perhaps you've already done it), you can remove the attribute in your repository

**Comment by NicolasGeraud** [comment]
I close this issue and refer to #354
